frontend/src/components/ui/Card/Card.test.tsx

```
@@ -0,0 +1,526 @@
+          <h2 id="card-title">Card Title</h2>
+        </Card>
+      );
+      expect(screen.getByTestId('card')).toHaveAttribute(
+        'aria-labelledby',
+        'card-title'
+      );
+    });
+
+    it('supports role attribute for semantic meaning', () => {
+      render(
+        <Card role="region" data-testid="card">
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<section>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)